runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -1633,6 +1613,28 @@ static TR::Register * generateMultianewArrayWithInlineAllocators(TR::Node *node,
    generateMemRegInstruction(TR::InstOpCode::SMemReg(use64BitClasses), node, generateX86MemoryReference(targetReg, TR::Compiler->om.offsetOfObjectVftField(), cg), classReg, cg);
    // Init 1st dim array size field
    generateMemRegInstruction(TR::InstOpCode::S4MemReg, node, generateX86MemoryReference(targetReg, fej9->getOffsetOfContiguousArraySizeField(), cg), firstDimLenReg, cg);
+   if (!TR::Compiler->om.compressObjectReferences())
+      { // Clear padding in contiguous array header layout. +4 because size field is 4 bytes wide.
+      generateMemImmInstruction(TR::InstOpCode::S4MemImm4, node, generateX86MemoryReference(targetReg, fej9->getOffsetOfContiguousArraySizeField() + 4, cg), 0, cg);
```

Comment:
Does the padding field exist for contiguous array header in 32-bit mode?  Looking at [`j9nonbuilder.h`](https://github.com/eclipse-openj9/openj9/blob/01b6a394d2850c9ea080068b2f5484576bdcd397/runtime/oti/j9nonbuilder.h), I don't think it does.  Sorry if I'm misreading it!